samples/hello-kapt/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+import org.gradle.api.internal.HasConvention
+import org.gradle.kotlin.dsl.delegateClosureOf
+import org.jetbrains.kotlin.gradle.plugin.KaptAnnotationProcessorOptions
+import org.jetbrains.kotlin.gradle.plugin.KaptJavacOptionsDelegate
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
+
+plugins {
+    application
+    idea
+    kotlin("jvm") version "1.1.61"
+    //Enable kapt plugin
+    kotlin("kapt") version "1.1.61"
+}
+
+application {
+    mainClassName = "samples.HelloWorldKt"
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile(kotlin("stdlib"))
+
+    //Use AutoValue to check that annotation processing works
+    //https://github.com/google/auto/tree/master/value
+    compileOnly("com.google.auto.value:auto-value:1.5")
+    //Kapt configuration for an annotation processor
+    kapt("com.google.auto.value:auto-value:1.5")
+}
+
+kapt {
+    // Type safe accessors for Kapt DSL
+    correctErrorTypes = true
+
+    // Example of Javac Options configuration
+    javacOptions(delegateClosureOf<KaptJavacOptionsDelegate> {
+        option("SomeJavacOption", "OptionValue")
+    })
+
+    //Kapt Arguments configuration
+    arguments(delegateClosureOf<KaptAnnotationProcessorOptions> {
+        arg("SomeKaptArgument", "ArgumentValue")
+    })
+}
+
+// Workaround for https://youtrack.jetbrains.com/issue/KT-17923 and related issues
```

Comment:
:x: While this may be valuable to people using "old" Kotlin versions I think we should not keep this around in the sample

Reply:
Okay,of course. I prepared this sample in times of Kotlin 1.1.51